articles/cognitive-services/Speech/API-Reference-REST/BingVoiceRecognition.md

```diff
@@ -78,6 +78,11 @@ The following characteristics are typical of dictation mode applications:
 * Users often plan ahead of time what they want to say and use more formal language.
 * Users employ full sentences lasting 5-8 seconds.
 
+> [!NOTE]
+> In Dictation & Conversation modes partial results are not returned. More stable, phrase results are.
```

Comment:
replace & with "and"
"More stable phrase results" might be misleading. I think you should just say 
are not returned. Only final phrase results are returned.